documentation/src/main/asciidoc/topics/ref_jgroups_inheritance_attributes.adoc

```diff
@@ -0,0 +1,34 @@
+[id='jgroups_inheritance_attributes-{context}']
+= Inheritance Attributes
+When you extend a JGroups stack, inheritance attributes let you adjust protocols and properties in the stack you are extending.
+
```

Comment:
We should mention that without such attributes the protocol is replaced completely

Reply:
only the changed attributes are overwritten since `stack.combine` defaults to COMBINE.